build.assets/Makefile

```diff
@@ -202,9 +203,10 @@ build-in-container: buildbox
 		   -e "GRAVITY_VERSION=$(GRAVITY_VERSION)" \
 		   -e "GRAVITY_TAG=$(GRAVITY_TAG)" \
 		   -e 'GRAVITY_LINKFLAGS=$(GRAVITY_LINKFLAGS)' \
+		   -e 'TF_BINARIES=$(TF_BINARIES)' \
```

Comment:
We should do the same with `$(BINARIES) ` and do away with literal paths (i.e. `$(LOCAL_GRAVITY_BUILDDIR)/tele`) in the `make` invocation.